Makefile

```
@@ -102,13 +102,18 @@ addcopy:
 	@scripts/add-copy.sh
 
 .PHONY: generate
-generate: gen-docs gen-mocks gen-code ## Generate docs, mocks, code, all auto generated assets
+generate: gen-docs gen-mocks gen-code gen-l1-commands ## Generate docs, mocks, code, L1 commands, all auto generated assets
 
 .PHONY: gen-code
 gen-code: ## Generate code
 	@echo "==> Generating code"
 	go run ./tools/cli-generator
 
+.PHONY: gen-l1-commands
+gen-L1-commands: ## Generate L1 commands
```

I find the L1 naming something not easy to navigate, especially by external contributors which would not have context
```suggestion
gen-api-commands: ## Generate commands under `atlas api`
```

---

I've renamed it